src/templates/Templates.js

```diff
@@ -450,18 +455,21 @@ const StateTemplates = {
   Procedure: {
     type: "Procedure",
     codes: [{...TypeTemplates.Code.Snomed}],
-    duration: {
-      low: 30,
-      high: 30,
-      unit: 'minutes'
-    }
+    distribution: {
+      "kind": "UNIFORM",
+      "parameters": {
+        "high": 60,
```

Comment:
You changed the high from 30 to 60 mins - deliberate?

Reply:
Yes. Since the values represent a range, I wanted them to be different.